src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,94 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75F;
+    private Node<K, V>[] nodes;
+    private int size;
+    private double threshold;
+
+    public MyHashMap() {
+        nodes = (Node<K, V>[]) new Node[DEFAULT_CAPACITY];
+        threshold = LOAD_FACTOR * DEFAULT_CAPACITY;
+        size = 0;
+    }
+
+    private class Node<K, V> {
+        private Node<K, V> next;
+        private K key;
+        private V value;
+        private int hash;
+
+        private Node(Node<K, V> next, K key, V value, int hash) {
+            this.next = next;
+            this.key = key;
+            this.value = value;
+            this.hash = hash;
+        }
+    }
```

Comment:
I'm not sure about the position of private inner classes, maybe they should be at the end of class. Can't find info about this.

Reply:
i dont know too
in all homeworks i did the same

Reply:
better move inner class at the end of main class) firstly because it is private, and all private should be after public
https://mate-academy.github.io/style-guides/java/java.html#s3.4.2-ordering-class-contents